tests/gradients/parameter_shift/test_parameter_shift_hessian.py

```diff
@@ -1507,12 +1499,12 @@ def circuit(params):
             return qml.probs([2, 3])
 
         params = np.array([0.5, 0.5, 0.5], requires_grad=True)
-        circuit(params)
 
         result = qml.gradients.param_shift_hessian(circuit)(params)
         assert np.allclose(result, np.zeros((3, 3, 4)), atol=0, rtol=0)
 
-        tapes, _ = qml.gradients.param_shift_hessian(circuit.qtape)
+        tape = qml.workflow.construct_tape(circuit)(params)
+        tapes, _ = qml.gradients.param_shift_hessian(tape)
```

Comment:
```suggestion
        tapes, _ = qml.gradients.param_shift_hessian(qml.workflow.construct_tape(circuit)(params))
```

Might be worth combining into a single line (at least in this case where it's still easy to read)

Reply:
True! Addressed in latest commit, resolving.